app/models/story_distributions/episode_distribution.rb

```diff
@@ -94,6 +94,7 @@ def episode_attributes
       content: story.description_html,
       categories: story.tags,
       published_at: story.published_at,
+      released_at: story.released_at,
```

Comment:
Note to self: Feeder needs to know about this new POST attribute.

Reply:
Updated note to self: apparently feeder doesn't care if you POST an extra attribute.  It just ignores it.

Reply:
Yes, that's the design, it is not using some schema to validate, so fine to update CMS first, then feeder.

Reply:
⚠️ Change request - I believe you also need to add `released_at` to the story representer, so async messages and api responses include it as well.